Common/Data/Custom/Quiver/QuiverPoliticalBeta.cs

```diff
@@ -0,0 +1,165 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using NodaTime;
+using ProtoBuf;
+using static QuantConnect.StringExtensions;
+
+namespace QuantConnect.Data.Custom.Quiver
+{
+    /// <summary>
+    /// Political beta for the specified company
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public class QuiverPoliticalBeta : BaseData
+    {
+
+        /// <summary>
+        /// The date of the follower count
+        /// </summary>
+        [ProtoMember(10)]
+        [JsonProperty(PropertyName = "Date")]
+        private string dateJson { get; set; }
+        [JsonIgnore]
+        public DateTime Date {
+            get {
+                Console.WriteLine(dateJson);
+                return DateTime.ParseExact(dateJson, "yyyy-MM-dd", null);
+                    }
+            set {
+                dateJson = value.ToStringInvariant("yyyy-MM-dd");
+            }
+        }
+
+        /// <summary>
+        /// The ticker of the company
+        /// </summary>
+        [ProtoMember(11)]
+        [JsonProperty(PropertyName = "Ticker")]
+        public string Ticker { get; set; }
+
+        /// <summary>
+        /// Correlation between daily excess returns and daily changes in Trump election odds
+        /// </summary>
+        [ProtoMember(12)]
+        [JsonProperty(PropertyName = "TrumpBeta")]
+        public decimal? TrumpBeta { get; set; }
+
+        /// <summary>
+        /// Trump's odds of winning the 2020 Presidential election, based on PredictIt betting markets
+        /// </summary>
+        [ProtoMember(14)]
+        [JsonProperty(PropertyName = "TrumpOdds")]
+        public decimal? TrumpOdds { get; set; }
```

Comment:
Let's abstract this class into a `PredictionBeta` class, with the following data model:

```C#
class QuiverPredictionBeta
{
    DateTime Date;
    
    // You can set these attributes manually by storing the data with the required tags for these
    string EventName;
    string FirstEventName;
    string SecondEventName;

    decimal FirstEventBeta;
    decimal SecondEventBeta; // Inverse of First Event Beta, can be calculated at runtime (i.e. with a function)

    decimal FirstEventOdds;
    decimal SecondEventOdds; // (1 - FirstEventOdds 😁)
}
```

This will enable better reusability in case we (or you) source more data from PredictIt 🙂